### PR TITLE
Fixed documentation and command for adding credits to a user's account for local development.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

While reviewing #10694, I needed to add credits to my account. I went to the documentation and copy pasted the commands, but it gave a syntax error in the rails console. This PR fixes the commands to run in the rails console to add credits to a user's account.

## QA Instructions, Screenshots, Recordings

Before

image

After

image
image

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [x] docs.forem.com
- [ ] readme
- [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

A video game character snowboarding

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -65,11 +67,12 @@ Boom, you have a new follower!
 If you ever want to add Listings locally, you must have credits on your account
 to "pay" for listing. Here's how:
 
-1. Open the rails console `rails c`.
+1. Open the rails console `rails console`.
```
Comment (nickytonline) [inline docs/faqs.md:70]: The actual fix in the docs is here. Everything above that changed is prettier formatting in action.

**Review by benhalpern — APPROVED** [review]

File: docs/faqs.md [hunk 2]
```diff
@@ -29,11 +29,13 @@ to your own `.env` file.
...
 On the home Feed, we only show comments above certain "score". It's likely the
-comments in the local environment will never meet this score. If you want to see comments locally, you will need to
-update the score of your local comments manually. Here's how:
+comments in the local environment will never meet this score. If you want to see
+comments locally, you will need to update the score of your local comments
+manually. Here's how:
 
 1. Open the terminal.
-2. Run `psql PracticalDeveloper_development` to open `psql`, the PostgreSQL terminal.
+2. Run `psql PracticalDeveloper_development` to open `psql`, the PostgreSQL
```
Comment (citizen428) [inline docs/faqs.md:37]: **suggestion (non-blocking):** Unrelated, but at one point we could change this to also mention `rails dbconsole`, which parses `config/database.yml`.

File: docs/faqs.md [hunk 3]
```diff
@@ -65,11 +67,12 @@ Boom, you have a new follower!
...
 to "pay" for listing. Here's how:
 
-1. Open the rails console `rails c`.
+1. Open the rails console `rails console`.
 2. Enter the following commands:
+
    ```ruby
-   user = User.find_by(username: "your_username")
-   Credit.add_to(user, 1000)
+   user = User.find_by(username: "your_username");
```
Comment (citizen428) [inline docs/faqs.md:74]: Prettier was over-eager here, there shouldn't be semi-colons in Ruby. Whatever the problem was, this is not the fix, semicolons in Pry just surpress showing output.

**Review by citizen428 — CHANGES_REQUESTED** [review]
Prettier was over-eager and generated some non-idiomatic Ruby.

File: docs/faqs.md (hunk 3 above)
Comment (nickytonline) [reply docs/faqs.md:74]: Sounds good. Thanks. I added the `;`, so that's on me. 😄

File: docs/faqs.md (hunk 2 above)
Comment (nickytonline) [reply docs/faqs.md:37]: TIL 👏🏻

File: docs/faqs.md (hunk 3 above)
Comment (nickytonline) [reply docs/faqs.md]: @citizen428, the funny thing is now that I removed the semi-colons, it appears to be just prettier formatting in action. I wonder if the empty line between the code snippet and the text was what fixed this.

**Review by jacobherrington — APPROVED** [review]
:+1:

**Review by citizen428 — APPROVED** [review]

**Pull request merged**